docs/shared/jedi.yaml

```diff
@@ -0,0 +1,8 @@
+jedi:
+  execution:
+    mpicmd: srun
+    batchargs:
+      nodes: 1
+      walltime: 08:00:00
+    executable: "/path/to/jedi.exe"
+  run_dir: "/path/to/dir/run"
```

Comment:
I would have expected this example to be somewhat similar to the one used to test the driver in a real-world scenario.

EDIT: The one you use for test configuration for the driver would be a nice one to model.

Reply:
I updated this accordingly. I made the update to `files_to_copy` and `files_to_link` more generic for now. I'm not sure if we have good examples of what users might see for jedi.